gcpdiag/lint/gcf/warn_2021_001_cloudfunctions_deprecated_runtimes.py

```diff
@@ -12,8 +12,8 @@ def run_rule(context: models.Context, report: lint.LintReportRuleInterface):
   cloudfunctions = gcf.get_cloudfunctions(context)
   if not cloudfunctions:
     report.add_skipped(None, 'no functions found')
-  for _, cloudfunction in sorted(cloudfunctions.items()):
-    if cloudfunction.runtime in ['go111', 'nodejs8']:
+  for cloudfunction in cloudfunctions.values():
```

Comment:
Why remove the sorting? I think it's useful to have the reported resources sorted by name. That's the reason to use items() and not values()

Reply:
There is no need for items because keys are unused. If we want to sort alphabetically there is a cleaner way I provided with lambda.